identity/identity.go

```diff
@@ -40,6 +47,13 @@ func (lt State) IsValid() error {
 	return errors.New("identity state is not valid")
 }
 
+//type IdentifierCredential struct {
+//	Subject      string `json:"subject"`
+//	Provider     string `json:"provider"`
+//	AccessToken  string `json:"access_token"`
+//	RefreshToken string `json:"refresh_token"`
+//}
+
```

Comment:
why is this commented out?

Reply:
Upstream's work in progress I guess https://github.com/ory/kratos/blob/master/identity/identity.go#L50-L55